examples/ui/ui.rs

```diff
@@ -17,7 +17,7 @@ fn main() {
         // Only run the app when there is user input. This will significantly reduce CPU/GPU use.
         .insert_resource(WinitSettings::desktop_app())
         .add_systems(Startup, setup)
-        .add_systems(Update, mouse_scroll);
+        .add_systems(Update, (mouse_scroll, update_ui_scale));
```

Comment:
I'm not sure about updating this example: There is already a specific example to ui_scaling, with the same name I think, so this can be controversial. My idea for now is leaving as it is and add an example to this tool  on the dev_tools dir as follow up.